pnr/nextpnr/ecp5/meta.yaml

```diff
@@ -26,31 +28,31 @@ requirements:
     - fontconfig  [not win]
     - make        [not win]
     - cmake
-    - python    {{ python }}
     - boost     {{ boost_version }}
     - boost-cpp {{ boost_version }}
     - libboost  {{ boost_version }}
     - py-boost  {{ boost_version }}
+    - python {{ python_version }}
   host:
     - bison        [not win]
     - eigen        [not win]
     - flex         [not win]
     - pkg-config   [not win]
     - patchelf     [linux]
     - prjtrellis
-    - python    {{ python }}
     - boost     {{ boost_version }} [not win]
     - boost-cpp {{ boost_version }} [not win]
     - libboost  {{ boost_version }} [not win]
     - py-boost  {{ boost_version }} [not win]
+    - python {{ python_version }}
   run:
     - libgcc-ng     [linux]
     - libstdcxx-ng  [linux]
     - prjtrellis
     - {{ pin_compatible('python', min_pin='x.x', max_pin='x.x') }}
```

Comment:
Remove this dependency. It adds a Python requirement depending on which is used in `host` Python but it's redundant when used along `python {{ python_version }}`.

Reply:
Also remove the same requirement in nextpnr-generic and nextpnr-ice40.